homeassistant/components/frontend/www_static/manifest.json

```diff
@@ -14,6 +14,26 @@
      "src": "/static/favicon-384x384.png",
      "sizes": "384x384",
      "type": "image/png"
-    }
+    },
```

Comment:
We shouldn't add extra formats that look different. They should be the existing icon in the manifest.json